queue.go

```diff
@@ -33,6 +46,16 @@ type worker struct {
 	confirms chan amqp.Confirmation
 }
 
+func init() {
+	cfg = &config{
```

Comment:
Read and validate the variables here using the `Netflix/go-env` lib. Check this out: [this](https://github.com/Bidroombv/ugo-inventory/blob/master/cmd/api/config.go#L11-L16) and [this](https://github.com/Bidroombv/ugo-inventory/blob/master/cmd/api/config.go#L36-L40)

Reply:
done